Sources/NIOCore/NIOSendable.swift

```diff
@@ -71,4 +58,6 @@ final class UnsafeMutableTransferBox<Wrapped> {
         self.wrappedValue = wrappedValue
     }
 }
+#if swift(>=5.5) && canImport(_Concurrency)
+extension UnsafeMutableTransferBox: @unchecked Sendable {}
```

Comment:
The changes in `NIOSendable.swift` make `UnsafeTransfer` and `UnsafeMutableTransferBox` available in Swift 5.4. This allows us to unconditionally use them without code duplication and `#if`s.